docs/manifest.json

```
@@ -169,6 +169,16 @@
                   "path": "./admin/infrastructure/validated-architectures/3k-users.md"
                 }
               ]
+            },
+            {
+              "title": "Scale Testing",
+              "description": "Ensure your deployment can handle your organization's needs",
+              "path": "./admin/infrastructure/scale-testing.md"
+            },
+            {
+              "title": "Scaling Utilities",
+              "description": "Tools to help you scale your deployment",
+              "path": "./admin/infrastructure/scaling-utility.md"
```

```suggestion
              "path": "./admin/infrastructure/scale-utility.md"
```
Or we rename the file